test/utils/math/SafeCast.test.js

```diff
@@ -41,7 +41,7 @@ contract('SafeCast', async (accounts) => {
     });
   }
 
-  [8, 16, 32, 64, 96, 128, 224].forEach(bits => testToUint(bits));
+  Array(31).fill().map((_, i) => (i + 1) * 8).forEach(bits => testToUint(bits));
```

Comment:
This pattern shows up often. Let's add a helper function `range(start, end, step)` or something like it.